docs/documentation/concepts/index.md

```diff
@@ -27,8 +27,8 @@ Things do not have to be physical devices; they can also represent a web service
 Things expose their capabilities through **Channels**.
 Whether an installation takes advantage of a particular capability reflected by a Channel depends on whether it has been configured to do so.
 When you configure your system, you do not necessarily have to use every capability offered by a Thing.
-You can find out what Channels are available for a Thing by looking at the documentation of the Thing's **Binding**.
-Bindings can be thought of as software adapters, making Things available to your home automation system.
+
+**Bindings** are add-ons that provide a way to link Items to physical devices, and to abstract the specific communications requirements of that device into a more generic openHAB world of Things, Items and Channels.
```

Comment:
* "a way to link Items to physical devices, and to abstract the specific communications requirements" is a good way to further describe the concerns of a binding.
* "into a more generic openHAB world of Things, Items and Channels"
    * do not use openHAB in the ESH documentation
    * this is a very generic description. The sentence may just end with something like "... that device for the framework".

Reply:
Okay on not using OH.  Forgot that this is dual-purpose text.  I will correct this.  Okay - point taken regarding this being a generic description.  I am not sure about the word framework.  I will take a look at your suggestion in the context of the entire re-write.  Framework may be okay - just need to switch over to my editor now and incorporate all of your suggestions and see how it feels.